README.md

```diff
@@ -23,6 +23,7 @@ See something you think belongs here? Feel free to fork and send a PR!
 - [The Nim forum](http://forum.nim-lang.org/)
 ### Libraries
 - [html2karax](https://github.com/nim-lang-cn/html2karax) - Converts static html to Karax in Nim
+- [faster-than-csv](https://github.com/juancarlospaco/faster-than-csv#csv2karax) - Convert CSV to Karax table
```

Comment:
faster-than-csv is nice but why the Python dependency?

Reply:
It is for Python.

Reply:
Then it's not for Karax, is it?

Reply:
The same way some some people work doing C++ modules for Python,
some of us do Nim modules for Python.
Python cant do Frontend thats why.

Reply:
Meh I still don't get it but how about I simply accept it and move on...

Reply:
I understand is a "gray" area, is Ok if not merged too, 
I just think Karax needs more popularity, wherever it is used. Thanks. 🙂